package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "hato-logging-plugin",
+  "version": "0.0.1",
+  "description": "A plugin for logging messages passing through hato",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openrm/hato-logging-plugin.git"
+  },
+  "keywords": [
+    "hato",
+    "amqp",
+    "logging"
+  ],
+  "author": "Open Room Inc.",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/openrm/hato-logging-plugin/issues"
+  },
+  "homepage": "https://github.com/openrm/hato-logging-plugin#readme",
+  "dependencies": {
+    "hato": "0.0.8"
```

Comment:
it might be good to declare it as a peer dependency so it uses the same version as the host module, otherwise it might cause some unexpected issues